modules/ROOT/partials/partial_devtools.adoc

```diff
@@ -101,17 +81,15 @@ JavaScript, TypeScript
 |Starknet-go |Go |Chainlink |Nethermind | link:https://github.com/NethermindEth/starknet.go[starknet-go Github repository]
 |===
 
-[#starknet-devnet]
-== starknet-devnet, starknet-devnet-rs
+[#starknet-devnet-rs]
+== starknet-devnet-rs
 
 [discrete]
 === What is it?
 
-A devnet is a Starknet instance that you run as a local node, which enables much quicker development than is possible using testnet, as well as providing privacy prior to launching on testnet.
-
-SpaceShard originally wrote starknet-devnet in Python, but they are now actively developing a version in Rust, starknet-devnet-rs. For now, the Python-based version is more feature-rich, with the most notable feature being the ability to fork the network at a given block, so if that’s important to you, then you need to use the Python-based version. However, starknet-devnet-rs runs more quickly, and the developers are working to bring it to feature parity with the Python-based starknet-devnet.
+A devnet is a Starknet instance that you run as a local node. A devnet enables much quicker development than is possible using testnet, as well as providing privacy prior to launching on testnet.
 
-starknet-devnet-rs is the only version that is receiving new features.
+starknet-devnet-rs can, among other features, fork the network at a given block. starknet-devnet-rs runs more quickly than the previous, Python-based starknet-devnet.
```

Comment:
I would also add, "dumping the current state and loading it on startup for a faster environment setup".